src/routes/notifications/v1/notifications.controller.ts

```diff
@@ -170,6 +182,23 @@ export class NotificationsController {
     return safeV2Array;
   }
 
+  private validateTimestamp(timestamp: number): void {
+    const currentDate = new Date();
+    currentDate.setMinutes(currentDate.getMinutes() + 5);
+    const currentTimestampInSeconds = Math.floor(currentDate.getTime() / 1000);
+
+    const inputDate = new Date(timestamp * 1000);
+    inputDate.setMinutes(inputDate.getMinutes() + 5);
+    const inputTimestampInSeconds = Math.floor(inputDate.getTime() / 1000);
+
+    if (
+      currentTimestampInSeconds - inputTimestampInSeconds >
+      NotificationsController.REGISTRATION_TIMESTAMP_EXPIRY
+    ) {
+      throw new BadRequestException('The signature is expired!');
+    }
+  }
```

Comment:
We should not be checking this as it is handled by the Transaction Service.

Reply:
Well, Since we are no longer calling the TX service for registration, we should validate the signature on our end.